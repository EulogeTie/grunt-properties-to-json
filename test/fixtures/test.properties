this.is.test.one      =  This is test "1"
  this.is.test.two    =  This is test 2
this.is.another.test  =  This is another test\u0021
