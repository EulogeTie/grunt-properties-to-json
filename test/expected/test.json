{"this":{"is":{"test":{"one":"This is test \"1\"","two":"This is test 2"},"another":{"test":"This is another test!"}}}}
